src/Lockedfield.php

```diff
@@ -291,7 +291,7 @@ public function showForm($ID, array $options = [])
     public static function canCreate()
     {
         if (static::$rightname) {
-            return Session::haveRight(static::$rightname, UPDATE);
+            return Session::haveRight(static::$rightname, CREATE);
         }
         return false;
     }
```

Comment:
This method is probably useless now, as the default implementation can be used now.

Reply:
indeed